pkg/plugins/user.go

```diff
@@ -101,9 +86,22 @@ func createUser(fs vfs.FS, u schema.User, console Console) error {
 		Gid:      &gid,
 		Users:    u.Name,
 	}
-	updateGroup.Apply(etcgroup, false)
+	err = updateGroup.Apply(etcgroup, false)
+	if err != nil {
+		return errors.Wrap(err, "creating the user's group")
+	}
 
-	uid := 1000
+	// reload the group to get the generated GID
```

Comment:
This wouldn't be needed if the ParseGroup had a pointer receiver. I don't want to change that here though to avoid breaking other things. We can consider changing it in the future to avoid this second parsing of groups.